bin/ns-bundle

```diff
@@ -114,7 +117,7 @@ function getTnsVersion() {
     return new Promise((resolve, reject) => {
         const childProcess = spawn("tns", ["--version"], { shell: true });
 
-        childProcess.stdout.on("data", resolve);
+        childProcess.stdout.on("data", version => resolve(String(version)));
```

Comment:
I think I'd prefer `version.toString()` here to `String(version)`.
The latter looks a bit odd, just a personal taste though, although there are some minor differences between the two.